### DISCUSSION TITLE
What color wax is Gouda cheese most commonly wrapped in? 🧀

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1261] [Galaxy Trivia] [answered]
**Bonus:** Which number on the dartboard is counterclockwise from "1"? 🎯

**Comment by tomerf** [comment]
Gouda frequently come in yellow wax.

Bonus: In the dartboard, 1 is right after 20, at the top of the board ("12 o'clock").

**Reply by ParanoidUser** [reply]
Hmm, not exactly what I was expecting on the cheese side of the equation. 🧀🐁

**Comment by ParanoidUser** [comment]
@peymansf2000 / @p3rception, take your chance 🎲

**Reply by peymansf2000** [reply]
:)
thank you

**Comment by peymansf2000** [comment, marked as answer]
Gouda cheese is most commonly wrapped in red or yellow wax.
In general, red wax is more commonly associated with aged Gouda, while yellow wax is often used for younger, milder Gouda cheeses.

As for the bonus question, the number on the dartboard counterclockwise from "1" is "20".

**Reply by ParanoidUser** [reply]
Ding ding ding! 🔔 We have a winner! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/peymansf2000?achievement=galaxy-brain&tab=achievements) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉
